src/manager_service.py

```diff
@@ -151,6 +175,7 @@ def _setup_service_file(config_file: Path, log_file: Path) -> None:
         config_file: The configuration file for the service.
         log_file: The file location to store the logs.
     """
+    python_path = Path(os.getcwd()) / "venv"
     service_file_content = textwrap.dedent(
         f"""\
```

Comment:
Perhaps we can add this to the systemd service environment declaration, or perhaps a configuration directory might be the best practice according to this [stack overflow](https://serverfault.com/questions/413397/how-to-set-environment-variable-in-systemd-service) answer.

So perhaps something like:
```/etc/systemd/system/github-runner-manager.conf
PYTHONPATH={str(python_path)}
...<other configuration env vars that might be required>
```

and in the service unit file
```diff
+ EnvironmentFile=/etc/systemd/system/github-runner-manager.conf
```

The best case would be to rename the service dir `/etc/systemd/system/github-runner-manager.d`
and have the service unit file as well as the configuration in the same directory.

Reply:
This is a tmp hack, to make the current reactive process work.
This PYTHONPATH related code will be removed with the reactive process re-design.